src/exporter.h

```diff
@@ -0,0 +1,191 @@
+#ifndef EXPORTER_H_
+#define EXPORTER_H_
+
+#include <array>
+#include <cmath>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "backend.h"
+#include "daosoa.h"
+#include "default_force.h"
+#include "inline_vector.h"
+#include "math_util.h"
+#include "param.h"
+
+
+using std::fstream;
+using namespace std;
+
+namespace bdm {
+
+class Exporter {
+ public:
+  template <typename daosoa>
+  void ToFile(daosoa* cells, string filename) const {
+    // This function exports the cell positions into a file,
+    // where each line contains the 3D position of a cell in square brackets.
+    const size_t n_vectors = cells->vectors();
+
+    ofstream outfile;
+    outfile.open("../demo/" + filename);
```

Comment:
Please avoid hardcoding a directory.
This code fails if the binary gets executed from any other directory than `build`.
Also a generic export function should not have knowledge about the repository structure. This information must be given as parameter from the caller
-> `outfile.open(filename);`